### ISSUE TITLE
JS-T: Meta - Update url-parse to the latest version in all tutorials

### ISSUE THREAD

**Issue by relativityboy** [steemit/devportal-tutorials-js#125]
A security vulnerability was identified in versions of `url-parse` that we're using. 

- [x] Update all url-parse versions to latest if at all possible.
- [x] If not possible, say why in `# Results`

**Comment by roboza** [comment]
Currently only libraries that we use appear to be using the `url-parse` package. I have updated all `package-lock.json` files to include the latest version. If this is good we can merge the branch otherwise if not we can leave as is. We are not fixed on a specific version of any packages or `url-parse` so technically if someone runs `npm update` they will get the latest versions of the packages.Currently only libraries that we use appear to be using the `url-parse` package. I have updated all `package-lock.json` files to include the latest version. If this is good we can merge the branch otherwise if not we can leave as is. We are not fixed on a specific version of any packages or `url-parse` so technically if someone runs `npm update` they will get the latest versions of the packages.
